Paymetheus/ViewModels/StartupWizard.cs

```diff
@@ -313,11 +315,23 @@ public bool UsePublicEncryption
                     MessageBox.Show("Private passphrase is required");
                     return;
                 }
+                if (string.IsNullOrEmpty(PrivatePassphrase2))
+                {
+                    MessageBox.Show("Confirm private passphrase");
+                    return;
+                }
+                if (!string.Equals(PrivatePassphrase, PrivatePassphrase2))
```

Comment:
Not a big deal but != works here.  C# does have operator overloading and in this case == and != are overloaded for strings for value, not reference equality.